hdw-common/src/systems/instruments/src/EFB/Settings/Pages/AtsuAocPage.tsx

```diff
@@ -41,16 +40,11 @@ export const AtsuAocPage = () => {
   };
 
   const handleTrafficSourceChange = (entry: string) => {
-    const map = {"NONE": 0, "SIM":1, "VATSIM": 2};
+    const map = {"NONE": 0, "SIM":1, "VATSIM": 2, "IVAO": 3};
     setTrafficSource(entry);
     SimVar.SetSimVarValue('L:A32NX_TRAFFIC_SELECTOR_SOURCE', 'number', map[entry]);
   }
 
-  const handleTrafficDefaultChange = (entry: string) => {
```

Comment:
Why was this removed?

Reply:
The setting was not connected with anything - so i assumed this was a left over.
But i can reintegrate it no problem.

Reply:
thats not true